src/kubectl.ts

```diff
@@ -71,6 +83,13 @@ class KubectlImpl implements Kubectl {
                     disposable.dispose();
                 }
             });
+            this.context.host.onDidChangeConfiguration((change) => {
+                if (change.affectsConfiguration('vs-kubernetes')) {
```

Comment:
consolidate the two ifs into one check `if (change.affectsConfiguration('vs-kubernetes') && this.sharedTerminal) {`